common/src/crt_fd.rs

```diff
@@ -21,10 +20,31 @@ pub type Offset = libc::off_t;
 #[cfg(windows)]
 pub type Offset = libc::c_longlong;
 
+// copied from stdlib::os
```

Comment:
how about moving rather than copying? it seems stdlib::os can call this function also from common.

Reply:
fair, I was being lazy :P